guides/source/action_cable_overview.md

```diff
@@ -771,6 +771,14 @@ config.action_cable.log_tags = [
 For a full list of all configuration options, see the
 `ActionCable::Server::Configuration` class.
 
+Client-side logs can be enabled with the following:
```

Comment:
Lets move this to a new section called "### Client side logging" before `### Other Configurations`

```
### Client side logging

Client side logging is disabled by default. You can enable this by setting the `ActionCable.logger.enabled` to true.

...Code example here
```